frontend/components/IJzerboeren/Step2/Step2.tsx

```diff
@@ -30,9 +30,9 @@ export const Step2: FunctionComponent = () => {
         <div className="holonContentContainer" css={stepShadow}>
             <TwoColumnSimulationLayout bottomRuler={false} style={{minHeight: "50rem"}}>
                 <div className={rawHtml.Container}>
-                    <p>De gemeente wil graag verkennen hoe andere oplossingen zouden presteren voor de 50 nieuwbouw
-                        woningen. Zowel gas als een warmtenet kun je hier instellen om de (net)situatie te vergelijken
-                        met het warmtepomp scenario.</p>
+                    <p>Om de gemeente te helpen met het verkennen van de prestatie van de andere warmteoplossingen kun je
+                        hieronder de opties 'gas' en 'warmtenet met restwarmte' instellen en vergelijken met de 
```

Comment:
Single quotes moeten overal vervangen worden door `&apos;` (&apos;) of door paren van `&lsquo;`/&lsquo; en `&rsquo;`/&rsquo;

Dan zal de build slagen en kan het gemerged worden.